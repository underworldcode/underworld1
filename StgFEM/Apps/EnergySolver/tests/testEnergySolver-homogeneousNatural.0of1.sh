#!/bin/sh
##~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
##
## Copyright (C), 2003-2006, Victorian Partnership for Advanced Computing (VPAC) Ltd, 110 Victoria Street,
##	Melbourne, 3053, Australia.
##
## Primary Contributing Organisations:
##	Victorian Partnership for Advanced Computing Ltd, Computational Software Development - http://csd.vpac.org
##	Australian Computational Earth Systems Simulator - http://www.access.edu.au
##	Monash Cluster Computing - http://www.mcc.monash.edu.au
##	Computational Infrastructure for Geodynamics - http://www.geodynamics.org
##
## Contributors:
##	Patrick D. Sunter, Software Engineer, VPAC. (pds@vpac.org)
##	Robert Turnbull, Research Assistant, Monash University. (robert.turnbull@sci.monash.edu.au)
##	Stevan M. Quenette, Senior Software Engineer, VPAC. (steve@vpac.org)
##	David May, PhD Student, Monash University (david.may@sci.monash.edu.au)
##	Louis Moresi, Associate Professor, Monash University. (louis.moresi@sci.monash.edu.au)
##	Luke J. Hodkinson, Computational Engineer, VPAC. (lhodkins@vpac.org)
##	Alan H. Lo, Computational Engineer, VPAC. (alan@vpac.org)
##	Raquibul Hassan, Computational Engineer, VPAC. (raq@vpac.org)
##	Julian Giordani, Research Assistant, Monash University. (julian.giordani@sci.monash.edu.au)
##	Vincent Lemiale, Postdoctoral Fellow, Monash University. (vincent.lemiale@sci.monash.edu.au)
##
##  This library is free software; you can redistribute it and/or
##  modify it under the terms of the GNU Lesser General Public
##  License as published by the Free Software Foundation; either
##  version 2.1 of the License, or (at your option) any later version.
##
##  This library is distributed in the hope that it will be useful,
##  but WITHOUT ANY WARRANTY; without even the implied warranty of
##  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
##  Lesser General Public License for more details.
##
##  You should have received a copy of the GNU Lesser General Public
##  License along with this library; if not, write to the Free Software
##  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
##
## $Id: testEnergySolver-homogeneousNatural.0of1.sh 968 2007-10-23 07:53:39Z JulianGiordani $
##
##~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

TEST_SCRIPT=./VMake/executableTester.sh
until test -r ${TEST_SCRIPT} ; do
        TEST_SCRIPT=../${TEST_SCRIPT}
done
. ${TEST_SCRIPT}

runAndHandleSystemTestStdLocations "StGermain ../EnergySolver2D.xml ./homogeneousNaturalBCs.xml" "$0" "$@"
