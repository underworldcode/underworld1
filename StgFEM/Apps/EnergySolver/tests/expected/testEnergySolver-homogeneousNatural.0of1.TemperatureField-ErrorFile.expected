Timestep 0: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is outside a tolerance 0.025.
Timestep 5: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is outside a tolerance 0.025.
Timestep 10: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is outside a tolerance 0.025.
Timestep 15: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is outside a tolerance 0.025.
Timestep 20: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is outside a tolerance 0.025.
Timestep 25: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is outside a tolerance 0.025.
Timestep 30: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is outside a tolerance 0.025.
Timestep 35: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is within a tolerance 0.025.
Timestep 40: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is within a tolerance 0.025.
Timestep 45: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is within a tolerance 0.025.
Timestep 50: Total integrated value of 'TemperatureField-ErrorMagnitudeField' is within a tolerance 0.025.
